cli/command/cluster/create.go

```diff
@@ -21,33 +24,76 @@ func NewCreateCommand(c cli.Interface) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.IntVarP(&opts.workers, "workers", "w", 2, "Initial number of worker nodes")
-	flags.IntVarP(&opts.managers, "managers", "m", 3, "Intial number of manager nodes")
+	flags.IntVarP(&opts.managers, "managers", "m", 3, "Initial number of manager nodes")
```

Comment:
Yes, don't worry about the local options just yet -- please focus on the aws stuff for right now. I mentioned when I spoke to you all this is getting refactored (it's a wip) and asked you to just let me know if you have any issue with aws. Thanks!